src/web/app/pages-session/session-result-page/session-result-page.component.html

```diff
@@ -43,6 +46,11 @@ <h1>
 
 <tm-loading-retry [shouldShowRetry]="hasFeedbackSessionResultsLoadingFailed" [message]="'Failed to load results'" (retryEvent)="retryLoadingFeedbackSessionResults()">
   <div *tmIsLoading="isFeedbackSessionResultsLoading">
+    <div *ngIf="isFeedbackSessionResultsEmpty" class="mt-4">
+      <div class="alert alert-primary d-flex align-items-center justify-content-center" style="background-color: #D1ECF1;" role="alert">
```

Comment:
I am not sure if inline styles like`background-color` and `margin` are needed here.. @ccyccyccy can you help to verify?

Reply:
The `background-color` might be unnecessary, but without the `margin` the text will be a bit off.
![image](https://user-images.githubusercontent.com/58477697/126892398-8a427d32-bdc6-4617-a635-222656825e5e.png)

Reply:
OK, I figured out that the margin is not needed. I removed the `<p>` tag and it looks fine.